plugins/module_utils/network/ios/argspec/ospfv2/ospfv2.py

```diff
@@ -442,6 +439,22 @@ def __init__(self, **kwargs):
                             "type": "dict",
                         },
                         "passive_interface": {"type": "str"},
+                        "passive_interfaces": {
```

Comment:
Can we make this `mutually_exclusive` with `passive_interface`?

Reply:
updated